.gitignore

```diff
@@ -7,3 +7,4 @@ doc
 Meta
 *.pdf
 docs
+*run.r
```

Comment:
you don't need to change anything, but in the future it is better to avoid entries in gitignore that are specific to you (see <https://stackoverflow.com/questions/1753070/how-do-i-configure-git-to-ignore-some-files-locally>)